src/conn/conn_dhandle.c

```diff
@@ -726,8 +726,15 @@ __wt_conn_btree_apply(WT_SESSION_IMPL *session, const char *uri,
             WT_STAT_CONN_SET(session, checkpoint_handle_skipped, conn->ckpt_skip);
```

Comment:
Can we A->Z this list based on the stat name variable?

Reply:
Fixed.

Reply:
Fixed in [3f03206](https://github.com/wiredtiger/wiredtiger/pull/10220/commits/3f032065de9a48e907f75fb11ba8f6ca4ef7c472)